src/NotificationBar/NotificationBar.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { BarDiv } from './NotificationBar.styles';
+
+export type Props = {
```

Comment:
We should have a conversation at some point as to whether we prefer type vs interface